sn_transfers/src/wallet/watch_only.rs

```diff
@@ -187,38 +177,9 @@ impl WatchOnlyWallet {
     {
         for k in unique_pubkeys {
             self.keyless_wallet.available_cash_notes.remove(k);
```

Comment:
shall the correspondent cash_note to be removed here ?

Reply:
not yet as they haven't necessarily been _spent_ until confirmed.